images/vm-route-forge/internal/controller/route/route_controller.go

```diff
@@ -223,6 +223,14 @@ func (c *Controller) Run(ctx context.Context, workers int) error {
 		return fmt.Errorf("failed to synchronize routing rules at start: %w", err)
 	}
 
+	go func() {
+		wait.UntilWithContext(newCtx, func(_ context.Context) {
```

Comment:
UntilWithContext is not descriptive, add a comment that AddSubnetRoutesToBlackHole will be executed every minute until context cancelation.